docker-compose-dev.yml

```
@@ -0,0 +1,24 @@
+db:
```

I would also recommend to specify the version of the compose file explicitly and then stick to it's format, i. e.,

```suggestion
version: "3"
services:
  db:
```